pkgs/servers/openafs/1.6/module.nix

```diff
@@ -51,8 +51,8 @@ in stdenv.mkDerivation rec {
     license = licenses.ipl10;
     platforms = platforms.linux;
     maintainers = [ maintainers.z77z maintainers.spacefrogg ];
-    broken = versionOlder kernel.version "3.18" ||
-             versionAtLeast kernel.version "4.18";
+    broken = versionOlder kernel.version "3.18"
+             || stdenv.targetPlatform.isAarch64;
```

Comment:
Okay, this line breaks the manual evaluation. But I don't know, why...

Reply:
Please keep this line as it is and set `defaultText` in the manual to fix evaluation.